fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -1180,32 +1180,36 @@ private void computeColocateJoinInstanceParam(PlanFragmentId fragmentId, int par
                 }
             }
 
+            // set bucket for scanrange
```

Comment:
It is recommended to enrich the comments and give a practical example to illustrate what is stored in this nested type.